xrpl/models/transactions/__init__.py

```diff
@@ -28,6 +28,7 @@
 from xrpl.models.transactions.check_cash import CheckCash
 from xrpl.models.transactions.check_create import CheckCreate
 from xrpl.models.transactions.clawback import Clawback
+from xrpl.models.transactions.delete_oracle import OracleDelete
```

Comment:
ditto for these file names

Reply:
fixed in [df462ba](https://github.com/XRPLF/xrpl-py/pull/701/commits/df462ba5ca42b12e4ab754c9c6446b990264f6da)